### PR TITLE
Add `cert-manager` sub-chart as quick start

### PR DESCRIPTION
**Pull request by kezhenxu94**
Also unify the feature switch name to `enabled`.

### CODE REVIEW

File: chart/operator/templates/cert.yaml [hunk 1]
```diff
@@ -13,12 +13,15 @@
...
-{{- if .Values.webhook.enable }}
+{{- if .Values.webhook.enabled }}
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: {{ include "operator.fullname" . }}-serving-cert
   namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "1"
```
Comment (innerpeacez) [inline chart/operator/templates/cert.yaml:24]: Issuer
Certificate

Do different weights to distinguish the execution order?

Comment (kezhenxu94) [reply chart/operator/templates/cert.yaml]: Done

**Comment by innerpeacez** [comment]
Hi, Could we make a tools chart to do these things, tools chart to dependent on our oap, operator chart. @kezhenxu94

**Comment by kezhenxu94** [comment]
> Hi, Could we make a tools chart to do these things, tools chart to dependent on our oap, operator chart. @kezhenxu94

Can you elaborate? How would that tools chart work and what does it do?

**Comment by innerpeacez** [comment]
Let the tools chart do things that are not directly related to the oap or operator. I don't know if it's good.

**Comment by kezhenxu94** [comment]
> > Can you elaborate? How would that tools chart work and what does it do?
> 
> Mainly, I want to deploy other things that oap needs with `one-click` (easy).

Isn't that what the subchart does?

> Later, we can make our showcases into charts. 

I'm currently migrating showcase to Chart, and this patch is one of the things needed.

> For example, otel may need opentelemetry-collector charts and ksm charts. But if oap dependent on these charts, it feels inappropriate.

Yes, so we should put these dependencies in showcase.

In my opinion the Helm Chart dependencies should be something that MUST have in order to make the main Chart work, though it can be pre-installed and we would use `condition` for that dependency. But optional features (like OTEL) should not add dependencies into Chart.

**Pull request merged**